src/pages/en/guides/markdown-content.md

```diff
@@ -355,11 +355,18 @@ export default {
 
 ### Syntax Highlighting
 
-Astro comes with built-in support for [Prism](https://prismjs.com/) and [Shiki](https://shiki.matsu.io/). 
+Astro comes with built-in support for [Shiki](https://shiki.matsu.io/) and [Prism](https://prismjs.com/). 
 
-By default, Prism is enabled but no Prism stylesheet is included. (Choose from the available [Prism Themes](https://github.com/PrismJS/prism-themes) and see the [list of languages supported by Prism](https://prismjs.com/#supported-languages) for options and usage.)
+Shiki is enabled by default, preconfigured with the `github-dark` theme. This should provide instant syntax highlighting for all code fences (\`\`\`) used in a markdown (`.md`) file or the [built-in `<Markdown />` component](#markdown-component). What's more, the compiled output will be limited to inline `style`s without any extraneous CSS classes, stylesheets, or client-side JS.
 
```

Comment:
"should" makes me nervous! 😅 

Is it correct to say that whichever you have enabled, Prism or Shiki, controls the syntax highlighting in both the plain markdown and the Markdown component?

I would be more likely to start with WHAT is controlled, then say HOW Shiki does it (specifying it's the default). Maybe something like:

Astro comes with built-in support for [Shiki](https://shiki.matsu.io/) and [Prism](https://prismjs.com/) for instant syntax highlighting for all code fences (\`\`\`) used in a markdown (`.md`) file and in the [built-in `<Markdown />` component](#markdown-component). What's more, the compiled output will be limited to inline `style`s without any extraneous CSS classes, stylesheets, or client-side JS.

Shiki is enabled by default, preconfigured with the `github-dark` theme.

Reply:
Erm, that rewrite is somewhat correct. The "compiled output..." line is Shiki-specific. We'll likely need a separate explainer for Prism. Pushing up a rewrite momentarily.

Reply:
Gotcha! Make it accurate, for sure!

I just would prefer that we say as much that is generic/common first, THEN talk about Shiki. Do whatever is correct!

Reply:
Well hate to say it, but... almost nothing is in common between these 2 tools 😆 Only similarity is "they do syntax highlighting" as the section heading implies. It'll sadly take 2 different explainers.

Reply:
Ha! But, to clarify, whichever one you set **does** do **both** the fenced code in Markdown, and the Markdown component, right?

Even if the commonalities are short, I would be tempted to at least start the section something like: 

Astro comes with built-in support for using either [Shiki](https://shiki.matsu.io/) or [Prism](https://prismjs.com/) for instant syntax highlighting for all code fences (```) used in a markdown (.md) file and in the [built-in <Markdown /> component](https://github.com/withastro/docs/pull/237#markdown-component).

THEN, by all means, go wild with Shiki, and make Prism an afterthought. I think if we're mentioning that there are two options, (and, both can actually be used in the same project, given that you can use the Prism and Code components) then it should be clear upfront exactly what setting the config actually affects. (I'm assuming, both those two types of code mentioned above?)